src/sass/_style.scss

```diff
@@ -214,9 +214,8 @@ $elevation-level-2-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.15);
 }
 
 .institute-icon {
-  @extend .mr-1;
-  @extend .text-gray;
-
+  margin: calc(var(--pgn-spacing-spacer-base) * 0.25) !important;
```

Comment:
It looks like `.mr-1` is still listed in css utilities on the docs site https://paragon-openedx.netlify.app/foundations/css-utilities/

Does continuing to use `.mr-1` here break things?

Reply:
This change is good as-is (see https://github.com/openedx/frontend-app-authn/pull/1436#discussion_r2079869474 for context)